sample-queries/sample-queries.json

```diff
@@ -1679,6 +1679,65 @@
         "tip": "enable ExternalItem.Read.All",
         "skipTest": false
     },
+    {
+        "id": "26743d2a-2306-40f9-b3e8-3f857af69269",
+        "category": "Compliance (beta)",
+        "method": "GET",
+        "humanName": "List ediscoveryCases",
```

Comment:
Human name is the name users will see on GE to know what the query will do hence, should be in prose form e.g. List ediscovery cases

Reply:
... and usually starting with a lower case. See examples from the snippet below:

![image](https://user-images.githubusercontent.com/40403681/91153477-ed75f600-e6c8-11ea-803e-c4a76bd477b5.png)

The underlined texts are what will be the `humanName`

Reply:
I got the impression that the humanName was supposed to be the name that is visible in docs...

![image](https://user-images.githubusercontent.com/43222512/91211236-4e023480-e6c3-11ea-8e54-5089a264f1a3.png)